Gaurav.java

```diff
@@ -0,0 +1,30 @@
+import java.io.IOException;
+
+
+public class Gaurav {
+
+//	 private static int count = 0;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)